src/cmp/cmp.go

```diff
@@ -31,9 +31,11 @@ func Less[T Ordered](x, y T) bool {
 
 // Compare returns
 //
-//	-1 if x is less than y,
-//	 0 if x equals y,
-//	+1 if x is greater than y.
+// -1 if x is less than y,
+//
+//	0 if x equals y,
+//
+// +1 if x is greater than y.
```

Comment:
this is not correct - edge case of `go fmt`. best fix would probably be to move the next line left one space relative to its original position

Reply:
Please don’t reply on this GitHub thread. Visit [golang.org/cl/511395](https://go-review.googlesource.com/c/go/+/511395#message-38c5f335c9a6a4c1e10357c82a8855cdc41bffa2).

Reply:
nope, I am not doing that. I am signed in here, not there, so if you cant accept that then feel free to ignore my comments.

Reply:
@1268 not sure I m using Goland, and the doc of the function is being rendered as "-1" is rendered as 

- 1

Reply:
here is correct code:
https://go.dev/play/p/t0cnJA_mbIc

Reply:
<img width="314" alt="image" src="https://github.com/golang/go/assets/24226219/c5e1a9f9-5e0d-41c4-b2cd-32810b402341">

Reply:
Its not my job to troubleshoot problems with your editor. only to direct you to what the SPEC says:

https://go.dev/doc/comment

so regardless of any output of any editor, the code I have written is correct according to the spec.